frontend/tests/utils/helpers.ts

```diff
@@ -1,9 +1,18 @@
-export async function goToPage(url: string) {
+import { TEST_URL } from "tests/common/constants";
+import { getText } from "./selectors";
+
+export async function goToPage(url: string = TEST_URL) {
   await page.goto(url);
 }
 
+export async function login(url: string = TEST_URL) {
+  await goToPage(url);
+  await page.click(getText("Log In"));
+  await page.waitForNavigation({ url, waitUntil: "networkidle" });
+}
+
 export async function tryUntil(assert: () => void) {
-  const MAX_RETRY = 10;
+  const MAX_RETRY = 50;
```

Comment:
update retry times to `50` times, since @mbarrien was saying this is flaky in `rdev`. 

I'm assuming the flakiness is caused by not giving the dataset_asset URL enough time to be generated in `rdev`, so hopefully this resolves the problem. If not we can troubleshoot further 👌 

Thank you!